src/lib/utils/logger/Logger.ts

```diff
@@ -0,0 +1,50 @@
+import { ILogger, LogLevel } from './ILogger';
+
+export class Logger implements ILogger {
+	public level: LogLevel;
```

Comment:
Shouldn't this technically be private?

Reply:
It's useful for extensions of the `Logger`.